tagstudio/src/qt/ts_qt.py

```diff
@@ -1331,58 +1331,60 @@ def get_frame_contents(self, index=0, query: str = ""):
             index,
             len(self.frame_dict[query]),
         )
-
-    def filter_items(self, query: str = ""):
+    
+    def filter_items(self, query: str = "", override: bool = True):
         if self.lib:
```

Comment:
While you're touching the code already, it could be worth to flip this condition as well. There's no `else:` block, so instead of this:
```
if self.lib:
    # all the code
```

it could be done as this:

```
if not self.lib:
    return

# the same code except it's less indented now and you dont have to scroll 50 lines to see what will happen (spoiler: nothing) if the condition `if self.lib:` is not fulfilled 
```